documentation/modules/oauth/proc-oauth-authorization-keycloak-example.adoc

```diff
@@ -256,14 +283,31 @@ keytool -keystore /tmp/truststore.p12 -storetype pkcs12 -alias sso -storepass $S
 KAFKA_HOST_PORT=my-cluster-kafka-bootstrap:9093
 STOREPASS=storepass
 
-echo "Q" | openssl s_client -showcerts -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.crt
+echo "Q" | openssl s_client -showcerts -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.pem
+----
++
+The obtained `.pem` file is usually not one single certificate, but a certificate chain. You only have to provide the top-most issuer CA which is listed as last in the `/tmp/my-cluster-kafka.pem`.
+You can extract it manually or using the following command:
++
+.Example command to extract the top CA certificate in a certificate chain
+[source,shell,subs="+quotes"]
 ----
+split -p "-----BEGIN CERTIFICATE-----" /tmp/my-cluster-kafka.pem kafka-
+for f in $(ls kafka-*); do mv $f $f.pem; done
+cp $(ls kafka-* | sort -r | head -n 1) my-cluster-kafka-ca.crt
+----
++
+NOTE: The proper way to obtain the trusted CA certificate is from a trusted source, and not by using the `openssl` command.
+      For this example we assume the client is running in a pod in the same namespace where the Kafka cluster was deployed.
+      If the client accessed the Kafka cluster from outside the Kubernetes cluster you would have to first determine the bootstrap address.
+      In that case you can also get the cluster certificate directly from the Kubernetes Secret, no need for `openssl`.
+      See: https://strimzi.io/docs/operators/latest/configuring.html#assembly-accessing-kafka-outside-cluster-str for more information.
```

Comment:
I removed the external access certificate fetching and added additional comments and a link into NOTE section.
The link itself probably has to be done differently.